websub-ballerina/module.md

```diff
@@ -0,0 +1,71 @@
+## Package Overview
+
+This package contains an implementation of the W3C [**WebSub**](https://www.w3.org/TR/websub/) recommendation, which facilitates a push-based content delivery/notification mechanism between publishers and subscribers.
```

Comment:
This module ...

Reply:
shall we make the first sentence short and make it consistent with other modules.